src/docs/references/events/index.md

```diff
@@ -0,0 +1,414 @@
+  "source": "source/marketplace-api",
+  "resourceId": "5bbb2f6f-568f-470a-9949-a655e3f6ac46",
+  "resourceType": "listing",
+  "resource": {
+    "id": "5bbb2f6f-568f-470a-9949-a655e3f6ac46",
+    "title": "Peugeot eT101",
+    "description": "7-speed Hybrid",
+    "deleted": false,
+    "geolocation": {
+      "lat": 40.64542,
+      "lng": -74.08508
+    },
+    "createdAt": "2018-03-23T08:40:24.443Z",
+    "state": "published",
+    "availabilityPlan": {
+      "type": "availability-plan/day",
+      "entries": [
+        {
+          "dayOfWeek": "mon",
+          "seats": 1
+        },
+        {
+          "dayOfWeek": "tue",
+          "seats": 2
+        }
+      ]
+    },
+    "privateData": {
+      "externalServiceId": "abcd-service-id-1234"
+    },
+    "publicData": {
+      "address": {
+        "city": "New York",
+        "country": "USA",
+        "state": "NY",
+        "street": "230 Hamilton Ave"
+      },
+      "category": "road",
+      "gears": 22,
+      "rules": "This is a nice, bike! Please, be careful with it."
+    },
+    "metadata": {
+      "promoted": true
+    },
+    "price": {
+      "amount": 1590,
+      "currency": "USD"
+    },
+    "author": {"id": "5cf4c0eb-513f-419b-a8be-bdb6c14be10a"},
+    "marketplace": {"id": "9deec37c-b59c-4884-8f60-e4944335c327"},
+    "images": [
+      {"id": "209a25aa-e7cf-4967-89c3-0f09b2d482ff"},
+      {"id": "98e11f3b-ea22-4b1b-8549-e543ae241133"},
+      {"id": "ee1a647a-a751-43c7-90a4-48e94654f016"}
+    ]
+  },
+  "previousValues": {
+    "title": "old title",
+    "availabilityPlan": {
+      "type": "availability-plan/day",
+      "entries": [
+        {
+          "dayOfWeek": "mon",
+          "seats": 1
+        }
+      ]
+    },
+    "publicData": {
+      "address": {
+        "city": "New York",
+        "country": "USA",
+        "state": "NY",
+        "street": "222 Hamilton Ave"
+      },
+      "rules": null
+    },
+    "images": [
+      {"id": "98e11f3b-ea22-4b1b-8549-e543ae241133"},
+      {"id": "d12b8ebc-4df8-4bd0-9231-2f05691831a4"}
+    ]
+  },
+  "auditData": {
+    "userId": "5cf4c0eb-513f-419b-a8be-bdb6c14be10a",
+    "adminId": null,
+    "clientId": "69ea8198-201c-48c4-a3bb-78b38e4059b0",
+    "requestId": "4b66e510-22cb-47ca-953f-8a8377af2ed0"
+  }
+}
+```
+
+For all event types, except all `*/deleted` events, the `resource`
+attribute of the event contains the full data for the corresponding
+API resource **after** the event. The `resource` object contains keys
+corresponding to the attributes that the Integration API resource has,
+including it's `id`. For list of those attributes, see the Integration
+API reference for each resource type (e.g.
+[user](https://www.sharetribe.com/api-reference/integration.html#users),
+[listing](https://www.sharetribe.com/api-reference/integration.html#users),
+etc).
+
+In addition, when the resource has one or more 1-to-1 [TODO relationships]() with
```

Comment:
@bladealslayer I wasn't sure where you intended to link here so couldn't fix the missing link

Reply:
Just the generic explanation for relationships: https://www.sharetribe.com/api-reference/#including-related-resources